debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
@mike0042, I am of the view to not use any third-party or external software for the build process, so I haven't tried alien as yet.
w.r.t %exclude - I haven't found any equivalent macro that does it.